modules/ROOT/pages/syntax/operators.adoc

```diff
@@ -370,18 +370,18 @@ For example, `1 > b` and `1 < b` are both false when b is NaN.
 * *Ordering* of spatial values:
  ** `ORDER BY` requires all values to be orderable.
  ** Points are ordered after arrays and before temporal types.
- ** Points of different CRS are ordered by the CRS code (the value of SRID field). For the currently supported set of xref::syntax/spatial.adoc#cypher-spatial-crs[Coordinate Reference Systems] this means the order: 4326, 4979, 7302, 9157
+ ** Points of different CRS are ordered by the CRS code (the value of SRID field). For the currently supported set of xref::values-and-types/spatial.adoc#cypher-spatial-crs[Coordinate Reference Systems] this means the order: 4326, 4979, 7302, 9157
```

Comment:
When I read this, it felt like the first CRS needed to be plural, I don't know the etiquette for acronymns and plurality tbh, but thought I'd mention it :)